pkgs/applications/editors/jetbrains/readme.md

```diff
@@ -19,7 +19,7 @@ To test the build process of every IDE (as well as the process for adding plugin
  - This will update binary IDEs and plugins, and automatically commit them
  - Source builds need a bit more effort, as they **aren't automated at the moment**:
    - Find the build of the stable release you want to target (usually different for pycharm and idea, should have three components)
-   - I find this at https://jetbrains.com/updates/updates.xml (search for `product name="`, then `fullNumber`)
+   - Build number is avaliable at https://www.jetbrains.com/idea/download/other.html for idea and https://www.jetbrains.com/pycharm/download/other.html for pycharm
```

Comment:
Nit: maybe this would be more readable with a nested list?

```md
- You can find the build number at
  - idea: https://www.jetbrains.com/idea/download/other.html
  - pycharm: https://www.jetbrains.com/pycharm/download/other.html
```